include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -2825,6 +2825,13 @@ FUNCTION(ExceptionPersonality,
          EFFECT(NoEffect),
          UNKNOWN_MEMEFFECTS)
 
+FUNCTION(MemsetS, memset_s, C_CC, AlwaysAvailable,
```

Comment:
Perhaps this should be a stdlib-defined runtime function so different platforms can bring their own memset_s/memset_explicit/explicit_bzero/SecureZeroMemory

Reply:
Right. This makes sense